File: executable/python/basic-histogram.py
import plotly.plotly as py
from plotly.graph_objs import *

py.sign_in('test-runner', '9h29fe3l0x')

import numpy as np
x = np.random.randn(500)

data = Data([
    Histogram(
        x=x
    )
])

plot_url = py.plot(data, filename='basic-histogram')

File: executable/julia/basic-histogram.jl
using Plotly

using Plotly
Plotly.signin("test-runner", "9h29fe3l0x")

x = randn(500)

data = [
  [
    "x" => x, 
    "type" => "histogram"
  ]
]

response = Plotly.plot([data], ["filename" => "basic-histogram", "fileopt" => "overwrite"])
plot_url = response["url"]